src/vaadin-dialog-resizable-mixin.html

```diff
@@ -193,6 +193,40 @@
         window.removeEventListener('touchmove', this._resizeListeners.resize[direction]);
         window.removeEventListener('mouseup', this._resizeListeners.stop[direction]);
         window.removeEventListener('touchend', this._resizeListeners.stop[direction]);
+        const dim = this._getResizeDimensions();
+        this.dispatchEvent(
+          new CustomEvent('resize', {
+            detail: {
+              width: dim.overlay.width,
+              height: dim.overlay.height,
+              contentWidth: dim.content.width,
+              contentHeight: dim.content.height
+            }
+          })
+        );
+      }
+
+      _getResizeDimensions() {
+        const overlay = this.$.overlay;
+        const overlayStyles = getComputedStyle(overlay.$.overlay);
+        const contentStyles = getComputedStyle(overlay.$.content);
+        const verticalPadding =
+          parseInt(contentStyles.paddingTop) +
+          parseInt(contentStyles.paddingBottom) +
+          parseInt(contentStyles.borderTopWidth) +
+          parseInt(contentStyles.borderBottomWidth);
+
+        overlay.$.content.style.boxSizing = 'content-box';
+        overlay.$.content.style.height = `calc(100% - ${verticalPadding}px)`;
+
+        const dim = {
+          overlay: {width: overlayStyles.width, height: overlayStyles.height},
+          content: {width: contentStyles.width, height: contentStyles.height}
+        };
+
+        overlay.$.content.style.removeProperty('box-sizing');
+        overlay.$.content.style.height = '100%';
+        return dim;
```

Comment:
Why doesn't this return an object with the `width, height, contentWidth, contentHeight` properties so you wouldn't need to construct it (in the dispatchEvent) from this object that has `overlay.width, overlay.height, content.width, content.height`?

Reply:
True true, should have returned the correct object. Refactored now.